### ISSUE TITLE
make pygpsclient compatible with gpsd and ubxtool

### ISSUE THREAD

**Issue by schaefer01** [semuconsulting/PyGPSClient#144]
# PyGPSClient Feature Request Template

I'd like the graphical ability to view satellites in parallel with commanding detailed configuration by gpsd and ubxtool

Only one device can access the ublox port on a device and the combination gpsd and ubxtool provide more command-ability than pygpsclient but gpsd and ubxtool are command line and not graphical

A clear and concise description of what the problem is e.g. I'd like to be able to do this [...]

It would be nice to have pygpsclient request info from gpsd rather than from the device

**Describe alternatives you've considered**

 1. stop the gpds daemon, then run pygpsclient

 2. exit pygpsclient then start the gpsd daemon

**Would you be willing to contribute a test device?**

If the request relates to a specific u-blox device that is not currently supported, would you be 
willing to contribute a device to the project for testing purposes?

**Comment by semuadmin** [comment]
Hi @schaefer01,

Sorry, but that would involve a significant amount of (_unpaid_) work, both to implement and test and then to keep in line with any subsequent gpsd or ubxtool API updates. I simply don't have the time myself, but happy to consider contributions.

Out of interest, what specific satellite information would you want to see displayed? There may be other ways to achieve this natively.

I'll move this to the pygnssutils Discussion Forum for others to follow up on.
